server/server.go

```diff
@@ -223,60 +223,59 @@ func (s *Server) githubEvent(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var pingEvent *github.PingEvent
 	if r.Header.Get("X-GitHub-Event") == "ping" {
-		pingEvent = PingEventFromJSON(ioutil.NopCloser(bytes.NewBuffer(buf)))
+		if pingEvent := PingEventFromJSON(ioutil.NopCloser(bytes.NewBuffer(buf))); pingEvent != nil {
+			mlog.Info("ping event", mlog.Int64("HookID", pingEvent.GetHookID()))
```

Comment:
Info o Warn level?

Reply:
This was `info` level before, so, if you'd ask my opinion; I'd make this `trace` level :)